internal/core/src/expr/ITypeExpr.h

```diff
@@ -705,6 +706,34 @@ class CompareExpr : public ITypeFilterExpr {
     const proto::plan::OpType op_type_;
 };
 
+class GISFunctioinFilterExpr : public ITypeFilterExpr {
+ public:
+    GISFunctioinFilterExpr(ColumnInfo cloumn,
+                           GISFunctionType op,
+                           const std::string wkb)
+        : column_(cloumn), op_(op), wkb_(wkb){};
+    std::string
+    ToString() const override {
+        OGRGeometry* geo = nullptr;
+        OGRGeometryFactory::createFromWkb(
+            wkb_.data(), nullptr, &geo, wkb_.size());
+        if (!geo) {
+            return "";
+        }
+        return fmt::format(
```

Comment:
geo should be freed with [OGRGeometryFactory::destroyGeometry()](https://gdal.org/en/latest/doxygen/classOGRGeometryFactory.html#aae001086e26985d95c36ccd255a8c6d5) after use.
[OGRGeometryFactory::destroyGeometry()](https://gdal.org/en/latest/doxygen/classOGRGeometryFactory.html#aae001086e26985d95c36ccd255a8c6d5)